### PR TITLE
Add note context menu

### PR DESCRIPTION
**Pull request by v0l**
image

### CODE REVIEW

File: src/Element/NoteFooter.tsx [hunk 1]
```diff
@@ -111,27 +112,65 @@ export default function NoteFooter(props: NoteFooterProps) {
...
+        title: "Snort",
+        url: url
+      });
+    } else {
+      await navigator.clipboard.writeText(url);
+    }
+  }
+
+  function menuItems() {
+    return (
```
Comment (verbiricha) [inline src/Element/NoteFooter.tsx:141]: When we add note mention we could add another option here for copying the raw note id so you can paste it in a note for mentioning it.

File: src/Element/NoteFooter.tsx [hunk 2]
```diff
@@ -140,15 +179,16 @@ export default function NoteFooter(props: NoteFooterProps) {
             <FontAwesomeIcon icon={faReply} />
           </div>
         </div>
+        <Menu menuButton={<div className="reaction-pill">
```
Comment (verbiricha) [inline src/Element/NoteFooter.tsx:182]: Screenshot 2023-01-21 at 09 06 59

wdyt about removing a bit of x padding? looks a bit wide.

**Review by verbiricha — APPROVED** [review]
minor comments, lgtm :shipit:

File: src/Element/Note.css [hunk 3]
```diff
@@ -1,43 +1,55 @@
...
+  border-radius: 16px;
+}
+
+.note>.header img:hover, .note>.header .name>.reply:hover, .note .body:hover {
+  cursor: pointer;
 }
 
-.note > .note-creator {
+.note>.note-creator {
   margin-top: 12px;
```
Comment (verbiricha) [inline src/Element/Note.css:42]: I think we should pad this so is aligned with note body.

```suggestion
  margin-top: 12px;
  padding-left: 56px;
```

File: src/Element/NoteFooter.tsx (hunk 2 above)
Comment (v0l) [reply src/Element/NoteFooter.tsx:182]: image

File: src/Element/Note.css (hunk 3 above)
Comment (v0l) [reply src/Element/Note.css:52]: image

**Pull request merged**